client/client-multi/client-cpp/CMakeLists.txt

```diff
@@ -15,21 +15,76 @@
 #
 
 cmake_minimum_required(VERSION 2.8.8)
+
 project (Kaa-cpp)
+enable_language(CXX)
+
+# Path to CMake's scripts.
+set(CMAKE_MODULE_PATH 
+	 ${CMAKE_MODULE_PATH}
+	 ${CMAKE_CURRENT_SOURCE_DIR}/Modules
+    )
+
+#
+# Sets common compilation flags.
+# 
+# NOTE: Platform specific compilation flags should be set 
+# in the corresponding platform CMake script(s).
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+
+if(KAA_DEBUG_ENABLED)
+    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -ggdb")
+else()
+    set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -os -s")
```

Comment:
Please, replace the "-os" flag with "-Os" to be complied with compiler documentation.